README.md

```diff
@@ -98,11 +98,9 @@ delay = protocol.begin_api_request()
 sleep(delay)  # Wait for the specified delay, using whatever mechanism is appropriate for your environment
 
 # Send the API request.
-delay = GET_ALL_SETTINGS.send_request(protocol, None)
-sleep(delay)
-
```

Comment:
Maybe instead of having this and the comment two lines down say "...in your async environment", we just change the comment on line 100 to clarify that we're using asyncio and you can use something different.